index.html

```diff
@@ -9997,6 +9997,12 @@ <h5>Presentational Role Inheritance</h5>
           <li>The user interface makes the meaning and purpose of each state apparent.</li>
           <li>The user interface provides a separate method for controlling each state.</li>
         </ul>
+		<p>
+			User agents MUST calculate an implicit value for <sref>aria-level</sref> from the document structure if the author does not explicitly specify a value.
```

Comment:
Oh I guess this is related to your open question. Honestly, i think we might need to explicitly say how it is calculated, because "from the document structure" seems a little to vague to me.

For example, if the tree items are group with divs that don't have role group, I think we should ignore that structure, right? This sentence is not clear about that.

Reply:
@spectranaut I actually already addressed my open question in this PR 😅: https://github.com/w3c/aria/pull/2104

Which I think might also address your comment? Let me know if you think it needs different wording (though those comments probably belong on the other PR)